CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+#  Changelog
```

Comment:
There is already a changelog here: https://github.com/mongodb-labs/mongo-arrow/blob/main/bindings/python/docs/source/changelog.rst

Reply:
Thanks, merged